test/Templates.Test/Helpers/AspNetProcess.cs

```diff
@@ -48,9 +61,12 @@ public AspNetProcess(ITestOutputHelper output, string workingDirectory, string p
                     .WaitForExit(assertSuccess: true);
             }
 
+            _httpPort = 5000 + rnd.Next(2000);
+            _httpsPort = 44300 + rnd.Next(1000);
```

Comment:
The plan is for this to go away with https://github.com/aspnet/BasicMiddleware/issues/275